lib/jsdom/living/helpers/traversal.js

```diff
@@ -4,9 +4,9 @@ const { DOCUMENT_NODE } = require("../node-type");
 
 // All these operate on and return impls, not wrappers!
 
-exports.closest = (e, localName) => {
+exports.closest = (e, localName, namespaceURI = undefined) => {
```

Comment:
Is there actually any time we use this that it shouldn't be the HTML namespace?

Reply:
Yes, like the following WPT that'll get enabled by this PR:

https://github.com/w3c/web-platform-tests/blob/be8b3bbb1017519fa27bd90b8a50bf0b5b33aedf/html/semantics/tabular-data/attributes-common-to-td-and-th-elements/cellIndex.html#L27-L33

Reply:
Right, but, shouldn't all callers of closest() always pass the HTML NS?

Reply:
Do you mean the default value should be the HTML NS or am I understanding wrong?

Reply:
Well, it should just always check HTML NS, and never allow checking for anything else, I think.

Reply:
Didn't seem to break anything, so done.